win32/devdoc/threadpool_win32.md

```diff
@@ -293,12 +293,12 @@ MOCKABLE_FUNCTION(, int, threadpool_schedule_work_item, THANDLE(THREADPOOL), thr
 ### threadpool_destroy_work_item
 
 ```c
-MOCKABLE_FUNCTION(, void, threadpool_destroy_work_item, THANDLE(THREADPOOL), threadpool, THREADPOOL_WORK_ITEM_HANDLE, work_item_context);
+static void threadpool_destroy_work_item(THANDLE(THREADPOOL_WORK_ITEM) work_item_context);
 ```
 
 `threadpool_destroy_work_item` closes the `ptp_work` member variable in `work_item_context`and then frees the `work_item_context`
 
-**SRS_THREADPOOL_WIN32_05_014: [ ** If `threadpool` is `NULL`, `threadpool_destroy_work_item` shall fail and return a non-zero value. **]**
+**SRS_THREADPOOL_WIN32_05_014: [ ** [Deprecated] If `threadpool` is `NULL`, `threadpool_destroy_work_item` shall fail and return a non-zero value. **]**
 
 **SRS_THREADPOOL_WIN32_05_015: [** If `work_item_context` is `NULL`, `threadpool_destroy_work_item` shall fail and not do anything before returning. **]**
 
```

Comment:
I think this spec is obsolete (dispose of a THANDLE cannot be called with NULL by the contract of THANDLE)
/consider removing.

Reply:
Done